raiden/network/proxies/service_registry.py

```diff
@@ -140,16 +137,17 @@ def set_url(self, url: str) -> None:
             msg = "URL provided to service registry must be a valid HTTP(S) endpoint."
             raise BrokenPreconditionError(msg)
 
-        with log_transaction(log, "set_url", log_details):
-            gas_limit = self.client.estimate_gas(self.proxy, "latest", "setURL", url)
-            if not gas_limit:
-                msg = f"URL {url} is invalid"
-                raise RaidenUnrecoverableError(msg)
-
-            log_details["gas_limit"] = gas_limit
-            transaction_hash = self.client.transact(self.proxy, "setURL", gas_limit, url)
-            receipt = self.client.poll_transaction(transaction_hash)
-            failed_receipt = check_transaction_threw(receipt=receipt)
-            if failed_receipt:
-                msg = f"URL {url} is invalid"
-                raise RaidenUnrecoverableError(msg)
+        extra_log_details: Dict[str, Any] = {}
```

Comment:
Why even instantiate this, seems to be unused other than next line?

Reply:
typing